lib/tasks/github_usage_tracker.rake

```diff
@@ -0,0 +1,6 @@
+namespace :github_usage_tracker do
+  desc "Record a current datapoint of GitHub API usage"
+  task :record_datapoint => :environment do
+    GithubUsageTracker.record_datapoint
+  end
```

Comment:
I thought of that but figured that placing it on a Sidekick queue displaces time...but I'm hardly knowing what that means, really. If I do that, can I get the worker to go immediately? Given the crazy abilities of the TSDB and that this is a good testing ground to see if we can do things like this, I kinda wanted to try observations only a minute a part to start.